Browsable FAQs/browsable_faqs_with_media.md

```diff
@@ -0,0 +1,291 @@
+    )
+
+  index_menu_items = map(page_list_data.body.results, &[&1.title, &1.title])
+  message = 1
+
+  selected_content_name =
+    list("Select topic", FetchContent, index_menu_items) do
+      text("""
+      @selected_index_name
+      -----
+      Select the topic you are interested in from the list
+      """)
+    end
+end
+
+card FetchContent, then: DisplayContent do
+  selected_content_id =
+    filter(page_list_data.body.results, &(&1.title == selected_content_name))[0].id
+
+  content_data =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/@selected_content_id/",
+      headers: [
+        ["Authorization", "Token @config.items.contentrepo_token"]
+      ],
+      query: [["whatsapp", "true"], ["message", "@message"]]
+    )
+end
+
+card DisplayContent when content_data.body.has_children do
+  # If there are children, then give the user a list of options to choose from
+  parent_body =
+    if(
+      count(content_data.body.body.text) > 0,
+      content_data.body.body.text.value.message,
+      "Select an item"
+    )
+
+  parent_title = content_data.body.title
+  selected_content_id = find(page_list_data.body.results, &(&1.title == selected_content_name)).id
+
+  page_list_data =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/",
+      headers: [
+        ["Authorization", "Token @config.items.contentrepo_token"]
+      ],
+      query: [
+        ["child_of", "@selected_content_id"]
+      ]
+    )
+
+  content_children = map(page_list_data.body.results, &[&1.title, &1.title])
+
+  selected_content_name =
+    list("Select a topic", FetchContent, content_children) do
+      text("""
+      @parent_title
+      -----
+      @parent_body
+      """)
+    end
+end
+
+card DisplayContent when isnumber(content_data.body.body.next_message) do
+  next_prompt = "@content_data.body.body.text.value.next_prompt"
+  next_prompt = if(len(next_prompt) > 0, next_prompt, "Tell me more")
+  message = content_data.body.body.next_message
+
+  buttons(FetchContent: "@next_prompt") do
+    text("""
+    @content_data.body.title
+    @content_data.body.body.text.value.message
+    """)
+  end
+end
+
+card DisplayContent when count(content_data.body.related_pages) > 0 do
+  related_data =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/@selected_content_id/",
+      headers: [
+        ["Authorization", "Token @config.items.contentrepo_token"]
+      ]
+    )
+
+  related_pages = map(related_data.body.related_pages, &[&1.title, &1.title])
+
+  selected_content_name =
+    list("Select related page", SelectRelatedPage, related_pages) do
+      text("""
+      @content_data.body.title
+      @content_data.body.body.text.value.message
+      """)
+    end
+end
+
+card DisplayContent
+     when isnumber(content_data.body.body.text.value.media) and
+            isnumber(content_data.body.body.text.value.image),
+     then: FetchMedias do
```

Comment:
It doesn't look like we're using this card, along with the `FetchMedias` card? I don't think this will trigger because this is looking for a message with both media and an image at the same time. I think we can remove this and the `FetchMedias` card.

Reply:
@rudigiesler this pr is ready for relook.